test/specs/marked/marked.json

```diff
@@ -5,6 +5,12 @@
     "html": "<p><code>someone@example.com</code></p>",
     "example": 1
   },
+  {
+    "section": "Links",
+    "markdown": "Link: [constructor][].\n\n[constructor]: https://example.org/",
+    "html": "<p>Link: <a href=\"https://example.org/\">constructor</a>.</p>",
+    "example": 10
+  },
```

Comment:
Can you keep the example numbers in order?

Reply:
Sure, done. (I thought they were alphabetically ordered by section.)